exporter/otelarrowexporter/factory.go

```diff
@@ -35,14 +36,23 @@ func NewFactory() exporter.Factory {
 }
 
 func createDefaultConfig() component.Config {
-	batcherCfg := exporterhelper.NewDefaultBatcherConfig() //nolint:staticcheck
-	batcherCfg.Enabled = false
-
+	// These defaults are taken from the experimental setup used
+	// in the blog post covering Phase 1 performance results.  These
+	// were the defaults used in the concurrentbatchprocessor
+	queueCfg := exporterhelper.NewDefaultQueueConfig()
+	queueCfg.WaitForResult = true
+	queueCfg.BlockOnOverflow = true
+	queueCfg.Sizer = exporterhelper.RequestSizerTypeItems
+	queueCfg.NumConsumers = arrow.DefaultProducersPerStream * arrow.DefaultNumStreams
+	queueCfg.Batch = &exporterhelper.BatchConfig{
+		FlushTimeout: time.Second,
+		MinSize:      1000,
+		MaxSize:      1500,
+	}
```

Comment:
Looking for help, the Batch, WaitForResult, and BlockOnOverflow settings can't be overridden in this way, what can I do?